cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -119,10 +118,10 @@
         });
     }
 
-    &__bottom-rule {
+    &.m-image-text-50-50__bottom-rule {
```

Comment:
Fixed in 7f1b3d7. Although I'll have to open a new issue because my admin view doesn't have a way to test it.